### ISSUE TITLE
This is not an error but i need help. When trying to connect my htc desire 601 phone it says nothing pops up

### ISSUE THREAD

**Issue by Gameinator** [Genymobile/scrcpy#3746]
- [X] I have read the [FAQ](https://github.com/Genymobile/scrcpy/blob/master/FAQ.md).
 - [X] I have searched in existing [issues](https://github.com/Genymobile/scrcpy/issues).

**Environment**
 - OS: Windows
 - scrcpy version: [e.g. 1.12.1]
 - installation method: Windows download
 - device model: Htc desire 601
 - Android version: No idea

**Describe the bug**
I was going threw my drawer when i saw my old htc phone. When trying to unlock it i remebered that some of the screen is broken so unlocking it does not work. I am ust trying to get OTG mode so i can use the device and get old photos. I am not at all a tech guy and when ever i open up the cmd center it just says 
"scrcpy 1.25 <https://github.com/Genymobile/scrcpy>
INFO: Killing adb daemon (if any)...
ERROR: Could not find any USB device"
My phone is connected and windows is showing that its connected. Any help?

**Comment by rom1v** [comment]
https://github.com/Genymobile/scrcpy/blob/master/FAQ.md#hidotg-issues-on-windows

**Comment by Gameinator** [comment]
> https://github.com/Genymobile/scrcpy/blob/master/FAQ.md#hidotg-issues-on-windows

I tried some of those things and it not working. Still showing as not found. The problem is i cant go into the phone so i cant see if usb debugging is on

**Comment by Gameinator** [comment]
> https://github.com/Genymobile/scrcpy/blob/master/FAQ.md#hidotg-issues-on-windows

Well my problem is that it nots even seeing my phone as a usb device so it wont show me anything

**Comment by rom1v** [comment]
> it nots even seeing my phone as a usb device

Yes, that's the driver issue on Windows this [FAQ entry](https://github.com/Genymobile/scrcpy/blob/master/FAQ.md#hidotg-issues-on-windows) is about. Check the comments linked in that FAQ.
